### DISCUSSION TITLE
How to remove spin from a project?

### DISCUSSION THREAD

**Discussion by akulmehta** [serversideup/spin#126] [Q&A] [answered]
I started a project with Spin but it feels like it is overkill for what it is doing. I wanted to simplify the code base - is there a way to uninstall spin from a project? I love Spin but its just this project doesn't need it so I want to remove it.

**Comment by jaydrogers** [comment, marked as answer]
> I started a project with Spin but it feels like it is overkill for what it is doing

Can you elaborate more? I'd love to hear your feedback.

> is there a way to uninstall spin from a project?

You would just remove `serversideup/spin` from the composer.json or your package.json. We designed Spin to use it when you want it, and remove it when you dont 😃

If you want the files removed, just remove anything with `docker-compose.*`, `Dockerfile` and `.infrastructure`

**Reply by akulmehta** [reply]
Well, I want the app I am building to just run via Docker. This app is a private app, so it doesn't need things like Traefik or the ability to have zero downtime deployment. The overhead of having to install spin feels unnecessary in this case.

Don't get me wrong - as I mentioned I love Spin. It's just in this case its not required, IMO.
